src/journal/journal-send.c

```diff
@@ -345,7 +345,7 @@ static int fill_iovec_perror_and_send(const char *message, int skip, struct iove
 
                         memcpy(buffer, "MESSAGE=", 8);
 
-                        if (k > 0) {
+                        if (k > 1) {
```

Comment:
Actually this doesn't matter. `k` can be either 0 or 2 or something large, see above. So I think this commit should be dropped.

Reply:
Actually `k` can be either 0 or > 2
 `k` cannot be equal to 1 and 2

Reply:
So making it `if (k > 2)` would be more clear? It would also make future changes outside of the if statement less likely to cause an issue

Reply:
i don't think this should be changed. "k > 0" is easily understood as "non-empty", and that's what we actually mean here...

Reply:
removed until a potentially better solution is found